generator/templates/description.jinja2

```diff
@@ -8,14 +8,17 @@
  {%- if description and ( since or history ) %}
  *
  {%- endif %}
- {%- if deprecated %}
- * @deprecated
+ {%- if deprecated is sameas true %}
+ * @deprecated in SmartDeviceLink {{ since }}
+ {%- elif history %}
+ * @added in SmartDeviceLink {{ history[0].since }}
+ {%- elif since is defined %}
+ {%- if since %}
+ * @added in SmartDeviceLink {{ since }}
  {%- endif %}
- {%- if history %}
- * @history SDL {{ history }}
  {%- endif %}
- {%- if since %}
- * @since SDL {{ since }}
+ {%- if deprecated is sameas true and history %}
+ * @added in SmartDeviceLink {{ history[0].since }}
```

Comment:
The `history[0].since` value does not appear to get printed here if a struct or enum was deprecated. 

### Reproduce
In `SDLDisplayType` I see:
```
 * @deprecated in SmartDeviceLink 5.0.0
 * @added in SmartDeviceLink 
```
1.0.0 is missing after `@added in SmartDeviceLink`

This is the RPC spec entry for `DisplayType`:
```
<enum name="DisplayType" deprecated="true" since="5.0">
    <description>See DAES for further infos regarding the displays</description>
    <element name="CID"/>
    <element name="TYPE2" />
    <element name="TYPE5" />
    <element name="NGN" />
    <element name="GEN2_8_DMA" since="3.0" />
    <element name="GEN2_6_DMA" since="3.0" />
    <element name="MFD3" since="2.0" />
    <element name="MFD4" since="2.0" />
    <element name="MFD5" since="2.0" />
    <element name="GEN3_8-INCH" internal_name="GEN3_8_INCH" since="3.0" />
    <element name="SDL_GENERIC" since="4.0" />
    <history>
        <enum name="DisplayType" since="1.0" until="5.0"/>
    </history>
</enum>
```